OJP/OJP_JourneySupport.xsd

```diff
@@ -216,7 +221,7 @@
 	</xs:group>
 	<xs:group name="PrivateServiceGroup">
 		<xs:annotation>
-			<xs:documentation>specific service operated by a Private Mode</xs:documentation>
+			<xs:documentation>specific service operated by a Private Mode. May also be a school bus, a company shuttle or some other transport opportunity for a (normally) closed community.</xs:documentation>
```

Comment:
Proposal:
Specialized service operated by a Private Mode, e.g. a school bus, corporate shuttle, or other transportation for a (usually) closed community.

Reply:
included